Makefile

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2022 2020-present Open Networking Foundation <info@opennetworking.org>
```

Comment:
@ray-milkey it seems weird to have the year `2022` **AND** the range `2020-present`. Looking at something like `onos-topo` it only has the range. Same for all other files here